src/services/db/index.ts

```diff
@@ -1,9 +1,10 @@
 import AWS = require("aws-sdk");
+import config from "../../config";
 
 AWS.config.update({
   region: "us-west-2",
   // @ts-ignore
-  endpoint: "https://dynamodb.us-west-2.amazonaws.com" // "http://localhost:8000"
+  endpoint: config.AWS_ENDPOINT

```

Comment:
After exploring this line and the original reason behind its `@ts-ignore` while talking on Discord, I was under the impression this should be changed to update `endpoint` key for the `AWS.Config` object: `AWS.config.dynamodb`. Does that still sound right or did you arrive at a different conclusion in your development process?
```suggestion
  dynamodb: { endpoint: config.AWS_ENDPOINT }
```
----------------
If the suggested changed to the object passed to `.update()` doesn't seem correct after removing the `@ts-ignore`, maybe the following is more correct:
```js
AWS.config.update({
  region: "us-west-2"
});
AWS.config.dynamodb = { endpoint: config.AWS_ENDPOINT };
```

Let me know if you've gleaned anything new since our chat. It seems that without nitpicking about this and its `@ts-ignore`, what you have here works (from what I gather). No big deal if it stays as is. :)